services/aur/aur.tester.js

```diff
@@ -16,7 +15,7 @@ t.create('version (valid)')
   .get('/version/yaourt.json?style=_shields_test')
   .expectJSONTypes(
     Joi.object().keys({
-      name: 'aur',
+      name: 'version',
```

Comment:
I'm somewhat confused here. Why has the label changed to `version`?

Reply:
Good question. Looking over the examples we usually use the platform name as the label on version badges so I shouldn't have changed this.